### ISSUE TITLE
rio ps -q option returns error

### ISSUE THREAD

**Issue by izaac** [rancher/rio#616]
**Describe the bug**
`rio ps` has a `-q` option to only display names.
```bash
rio ps --help
NAME:
   rio ps - List services

USAGE:
   rio ps [OPTIONS]

DESCRIPTION:
   To view all rio services, run `rio ps`

OPTIONS:
   --quiet, -q     Only display Names
   --format value  'json' or 'yaml' or Custom format: '{{.Name}} {{.Obj.Name}}' [$FORMAT]
   --all, -a       print all resources, including router and externalservice
```

But it returns: FATA[0000] template: :1:2: executing "" at <.Name>: can't evaluate field Name in type ps.ServiceData

**To Reproduce**
1. install rio, spin up a service.
2. do a `rio ps -a` or `rio ps -q -a` and just a `rio ps -a`
3. `rio ps -a` works only, using `-q` returns FATA error.

**Expected behavior**

Returned output as described in `rio ps --help`

**Kubernetes version & type (GKE, on-prem)**: `kubectl version`
```
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2-k3s.1", GitCommit:"b8b17ba55f20e590df507fce333dfee13ab438c6", GitTreeState:"clean", BuildDate:"2019-10-16T05:17Z", GoVersion:"go1.13.3", Compiler:"gc", Platform:"linux/amd64"}
```
Type:
**Rio version**: `rio info`
```
rio version v0.6.0-alpha.1 (7678d67d)
```

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha2.

**Comment by rancher-max** [comment]
Fixed in v0.6.0-alpha.2
